Linking the two .cpp files results in duplicate definition errors

## Issue Details

I have placed the two files kinetic_partition.cpp and ksr_basic.cpp in the same folder of my project (with the CGAL environment properly configured). However, during linking, the linker reports errors about duplicate definitions of the same symbols. The full error messages are as follows:

1>ksr_basic.obj : error LNK2005: "class std::tuple<unsigned char,unsigned char,unsigned char> const __cdecl CGAL::KSP_3::internal::get_idx_color(unsigned __int64)" (?get_idx_color@internal@KSP_3@CGAL@@YA?BV?$tuple@EEE@std@@_K@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_polygon(class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_polygon@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_polygona(class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_polygona@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_polygons(class std::vector<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > >,class std::allocator<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > > > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_polygons@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@V?$allocator@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@@2@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_polygons(class std::vector<class std::vector<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > >,class std::allocator<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > > >,class std::allocator<class std::vector<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > >,class std::allocator<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > > > > > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_polygons@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$vector@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@V?$allocator@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@@2@@std@@V?$allocator@V?$vector@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@V?$allocator@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@@2@@std@@@2@@std@@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_indexed_triangles(class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > const &,class std::vector<unsigned __int64,class std::allocator<unsigned __int64> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_indexed_triangles@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@AEBV?$vector@_KV?$allocator@_K@std@@@5@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_indexed_polygons(class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > const &,class std::vector<class std::vector<unsigned __int64,class std::allocator<unsigned __int64> >,class std::allocator<class std::vector<unsigned __int64,class std::allocator<unsigned __int64> > > > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_indexed_polygons@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@AEBV?$vector@V?$vector@_KV?$allocator@_K@std@@@std@@V?$allocator@V?$vector@_KV?$allocator@_K@std@@@std@@@2@@5@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_polygons(class std::vector<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > >,class std::allocator<class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > > > const &,class std::vector<class CGAL::IO::Color,class std::allocator<class CGAL::IO::Color> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_polygons@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@V?$allocator@V?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@@2@@std@@AEBV?$vector@VColor@IO@CGAL@@V?$allocator@VColor@IO@CGAL@@@std@@@5@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj
1>ksr_basic.obj : error LNK2005: "void __cdecl CGAL::KSP_3::internal::dump_points(class std::vector<class CGAL::Point_3<class CGAL::Epick>,class std::allocator<class CGAL::Point_3<class CGAL::Epick> > > const &,class std::vector<class CGAL::Vector_3<class CGAL::Epick>,class std::allocator<class CGAL::Vector_3<class CGAL::Epick> > > const &,class std::vector<class CGAL::IO::Color,class std::allocator<class CGAL::IO::Color> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?dump_points@internal@KSP_3@CGAL@@YAXAEBV?$vector@V?$Point_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Point_3@VEpick@CGAL@@@CGAL@@@std@@@std@@AEBV?$vector@V?$Vector_3@VEpick@CGAL@@@CGAL@@V?$allocator@V?$Vector_3@VEpick@CGAL@@@CGAL@@@std@@@5@AEBV?$vector@VColor@IO@CGAL@@V?$allocator@VColor@IO@CGAL@@@std@@@5@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@5@@Z) already defined in kinetic_partition.obj

I believe there are some issues with the header file implementation, and I hope these problems can be resolved here.

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Windows 64 bits
* Compiler: VS 2022
* Release or debug mode:
* Specific flags used (if any):
* CGAL version:
* Boost version:
* Other libraries versions if used (Eigen, TBB, etc.):

--- a/Kinetic_space_partition/include/CGAL/KSP/debug.h
+++ b/Kinetic_space_partition/include/CGAL/KSP/debug.h
@@ -40,7 +40,7 @@ namespace CGAL {
 namespace KSP_3 {
 namespace internal {
 
-const std::tuple<unsigned char, unsigned char, unsigned char>
+inline const std::tuple<unsigned char, unsigned char, unsigned char>
 get_idx_color(std::size_t idx) {
 
   CGAL::Random rand(static_cast<unsigned int>(idx));
@@ -50,7 +50,7 @@ get_idx_color(std::size_t idx) {
     static_cast<unsigned char>(rand.get_int(32, 192)));
 }
 
-CGAL::IO::Color get_color(std::size_t idx) {
+inline CGAL::IO::Color get_color(std::size_t idx) {
   CGAL::Random rand(static_cast<unsigned int>(idx));
   return CGAL::IO::Color(rand.get_int(32, 192), rand.get_int(32, 192), rand.get_int(32, 192));
 }
@@ -929,7 +929,7 @@ void dump_polygon(const std::vector<typename K::Point_3>& pts, const std::string
   saver.export_polygon_soup_3(pts2, filename);
 }
 
-void dump_polygon(const std::vector<CGAL::Epick::Point_3>& pts, const std::string& filename) {
+inline void dump_polygon(const std::vector<CGAL::Epick::Point_3>& pts, const std::string& filename) {
   Saver<CGAL::Epick> saver;
   std::vector<std::vector<CGAL::Epick::Point_3> > pts2;
   pts2.push_back(pts);
@@ -937,7 +937,7 @@ void dump_polygon(const std::vector<CGAL::Epick::Point_3>& pts, const std::strin
   saver.export_polygon_soup_3(pts2, filename);
 }
 
-void dump_polygona(const std::vector<CGAL::Epick::Point_3>& pts, const std::string& filename) {
+inline void dump_polygona(const std::vector<CGAL::Epick::Point_3>& pts, const std::string& filename) {
   Saver<CGAL::Epick> saver;
   std::vector<std::vector<CGAL::Epick::Point_3> > pts2;
   pts2.push_back(pts);
@@ -945,30 +945,31 @@ void dump_polygona(const std::vector<CGAL::Epick::Point_3>& pts, const std::stri
   saver.export_polygon_soup_3(pts2, filename);
 }
 
-void dump_polygons(const std::vector<std::vector<CGAL::Epick::Point_3> >& pts, const std::string& filename) {
-  Saver<CGAL::Epick> saver;
-
-  saver.export_polygon_soup_3(pts, filename);
-}
-void dump_polygons(const std::vector<std::vector<std::vector<CGAL::Epick::Point_3> > >& pts, const std::string& filename) {
+inline void dump_polygons(const std::vector<std::vector<CGAL::Epick::Point_3> >& pts, const std::string& filename) {
   Saver<CGAL::Epick> saver;
 
   saver.export_polygon_soup_3(pts, filename);
 }
 
-void dump_indexed_triangles(const std::vector<CGAL::Epick::Point_3>& pts, const std::vector<std::size_t>& tris, const std::string& filename) {
+inline void dump_polygons(const std::vector<std::vector<std::vector<CGAL::Epick::Point_3> > >& pts, const std::string& filename) {
+  Saver<CGAL::Epick> saver;
+
+  saver.export_polygon_soup_3(pts, filename);
+}
+
+inline void dump_indexed_triangles(const std::vector<CGAL::Epick::Point_3>& pts, const std::vector<std::size_t>& tris, const std::string& filename) {
   Saver<CGAL::Epick> saver;
 
   saver.export_indexed_triangles_3(pts, tris, filename);
 }
 
-void dump_indexed_polygons(const std::vector<CGAL::Epick::Point_3>& pts, const std::vector<std::vector<std::size_t> >& polys, const std::string& filename) {
+inline void dump_indexed_polygons(const std::vector<CGAL::Epick::Point_3>& pts, const std::vector<std::vector<std::size_t> >& polys, const std::string& filename) {
   Saver<CGAL::Epick> saver;
 
   saver.export_indexed_polygons_3(pts, polys, filename);
 }
 
-  void dump_polygons(const std::vector<std::vector<CGAL::Epick::Point_3> >& pts, const std::vector<CGAL::IO::Color>& colors, const std::string& filename) {
+inline void dump_polygons(const std::vector<std::vector<CGAL::Epick::Point_3> >& pts, const std::vector<CGAL::IO::Color>& colors, const std::string& filename) {
   Saver<CGAL::Epick> saver;
 
   saver.export_polygon_soup_3(pts, colors, filename);
@@ -1013,7 +1014,7 @@ void dump_polygons(
   saver.export_polygon_soup_3(polygons, name);
 }
 
-void dump_points(const std::vector<CGAL::Epick::Point_3>& pts, const std::vector<CGAL::Epick::Vector_3>& normals, const std::vector<CGAL::IO::Color>& colors, const std::string& filename) {
+inline void dump_points(const std::vector<CGAL::Epick::Point_3>& pts, const std::vector<CGAL::Epick::Vector_3>& normals, const std::vector<CGAL::IO::Color>& colors, const std::string& filename) {
   Saver<CGAL::Epick> saver;
   saver.export_points_3(pts, normals, colors, filename);
 }

